src/shogun/kernel/GaussianKernel.cpp

```diff
@@ -24,7 +24,7 @@ GaussianKernel::GaussianKernel() : ShiftInvariantKernel()
 	m_distance=dist;
 
 	SG_ADD(
-	    &m_log_width, "log_width", "Kernel width in log domain",
+	    &m_width, "width", "Kernel width",
```

Comment:
would need a positive constraint

Reply:
yes, working on this to be able to pass constraint together with auto value. But I think for this to work well I will push the PR where I replace the SG_ADD macros with functions